### PR TITLE
Build Landing page, home page, login and signup page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
- Build the Landing page, 
- home page,
-  login
- signup page
- preview link: https://deploy-preview-1--todayspost.netlify.app/

### CODE REVIEW

File: src/pages/home-page/homepage.css [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+}
+
+.homepage-section {
+  display: flex;
+  margin-top: 1rem;
+  /* margin: 3rem 10rem; */
+  height: 100vh;
+  justify-content: center;
+  width: 70vw;
+  /* padding: 3rem 0; */
```
Comment (rohanmathur91) [inline src/pages/home-page/homepage.css]: remove commented code

File: src/pages/signup-page/SignUpPage.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    firstName: "",
+    lastName: "",
+  });
+  const [error, setError] = useState("");
+  let navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [type, setType] = useState("password");
+  const [confirmPasswordType, setConfirmPasswordType] = useState("password");
+  const [passwordError, setPasswordError] = useState("");
+  const doValidate = () => {
```
Comment (rohanmathur91) [inline src/pages/signup-page/SignUpPage.js:22]: you can put this function in utils or helper

**Review by rohanmathur91 — APPROVED** [review]
Code is readable and looks good to merge, just curious why you are not using any UI library or tailwind?

File: src/pages/signup-page/SignUpPage.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      setError("");
+    }
+    if (userData.password !== userData.confirmPassword) {
+      setPasswordError("Password does not match!");
+      return false;
+    } else {
+      setPasswordError("");
+    }
+    return true;
+  };
```
Comment (rohanmathur91) [inline src/pages/signup-page/SignUpPage.js]: you can put this function in utils or helper

Comment (ishitakeshawani) [reply src/pages/signup-page/SignUpPage.js]: done

File: src/pages/home-page/homepage.css (hunk 1 above)
Comment (ishitakeshawani) [reply src/pages/home-page/homepage.css]: done

**Comment by ishitakeshawani** [comment]
> Code is readable and looks good to merge, just curious why you are not using any UI library or tailwind?

I installed tailwind but its class was not working in the project. I spent the whole day and then I decided to skip it. :(

**Review by aniketsingh13 — APPROVED** [review]
UI look good
folder structure is also good
good to merge

**Pull request merged**